marimbabot_behavior/src/marimbabot_behavior/behavior_node.py

```diff
@@ -369,24 +387,6 @@ def callback_command(self, command_msg):
                     rospy.logwarn('The audio preview is already playing.')
                     self.response_pub.publish('The audio preview is already playing.')
 
-            elif command_msg.behavior == "stop":
```

Comment:
Why did you change this? @scaredycode

Reply:
To match the decision tree. It has no negative effects on the functionality. A positive effect is that we will no longer get the "No notes to play" Response for using the 'stop' command before a successful 'read'